app/code/Magento/CustomerGraphQl/Model/Customer/CreateCustomerAccount.php

```diff
@@ -96,6 +104,7 @@ public function execute(array $data): CustomerInterface
      */
     private function createAccount(array $data): CustomerInterface
     {
+        $this->validateData($data);
```

Comment:
Please, move this logic in separate class

Reply:
Done